### ISSUE TITLE
Suggestion: xtest should initialize the environment before starting test

### ISSUE THREAD

**Issue by airbak** [OP-TEE/optee_test#225]
When xtest failed or press "ctrl + c" to  stop it,  the secure storage test will remain some files.
If xtest start again, it will fail because the files already exist.
With REE FS,  we can  delete these files manually, but with RPMB FS, it is not easy to delete these files.
So, I suggest that xtest initialize the environment before starting test, such as deleting the file that might affect the test.

**Comment by jforissier** [comment]
Would building OP-TEE with `CFG_RPMB_RESET_FAT=y` help? I think it should. You would need to reboot the board to clear the RPMB data (just starting xtest again would not be enough), but it should work.

**Comment by airbak** [comment]
CFG_RPMB_RESET_FAT=y could help, but other files will also be deleted.
XTEST may be used to test a release version,  CFG_RPMB_RESET_FAT=y is not a good way to solve this issue.

**Comment by jforissier** [comment]
Fair enough ;-)
Since each TA is only allowed to access its own persistent objects, and since you want to be able to test a release OP-TEE, the only option I see is to modify all the TAs that create persistent objects so that they implement a command to enumerate and delete all their persistent objects. That is `ta/storage` and `ta/storage_benchmark`.

**Comment by jbech-linaro** [comment]
I'm closing the ticket, either because it has already been answered or that it is no longer relevant or it could be lack of response from the author. Having that said, feel free to re-open the ticket if you have more to add to the ticket.
